### PR TITLE
[CodeSec AG] Update Monitor and Fix Issues [RLP-92083]

### PR DESCRIPTION
**Pull request by anaghapamidi**
Summary of Changes:

Updated the existing Monitor and Fix Issues in Your Scans with latest flow with Projects 2.0 

Modified files:
-book.yml
- monitor-fix-issues-in-scan.adoc

Add new files:
- create-and-manage-code-category-views.adoc
- fix-issues-in-a-scan-result.adoc

### CODE REVIEW

**Comment by anaghapamidi** [comment]
awaiting on @gilad's review.

File: code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan/fix-issues-in-a-scan-result.adoc [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+In this example, on IaC Misconfiguration you see `aws_s3_bucket.bad_bucket` resource block with an issue that has a fix.
++
+image::proj-22.png[width=800]
+
+. Create a Pull Request with an issue fix.
+
+.. Select add corresponding to an issue and then view the issue in fix cart.
++
+In this example, you see `aws_s3_bucket.bad_bucket` in the fix cart.
++
```
Comment (GiladMark) [inline code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan/fix-issues-in-a-scan-result.adoc:30]: we should show in the documentation the fact that you click on an issue and then you see in the sidecar (resource explorer) the issue that was detected and the suggested fix

File: code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan/monitor-fix-issues-in-scan.adoc [hunk 2]
```diff
@@ -0,0 +1,373 @@
...
+|===
+
+| Projects Code Category View
+| Description
+
+|Overview
+|A summary view of all your scan results across all code categories. The Overview lists errors by prioritizing build issues across the integrations.
+
+|IaC Misconfiguration
+|The view lists security issues after scanning integrated repositories, wherein the master branch of a repository lists all code security violations in the resource block with code tags, resource dependencies and resource history. In this view, you can also identify Drift and trigger Manual Scans for your repository.
```
Comment (GiladMark) [inline code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan/monitor-fix-issues-in-scan.adoc]: let's call it "default" branch and not "master"

Comment (GiladMark) [reply code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan/monitor-fix-issues-in-scan.adoc]: trigger manual scans is across all views - this is an action from the upper menu

File: code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan/monitor-fix-issues-in-scan.adoc [hunk 3]
```diff
@@ -0,0 +1,373 @@
...
+
+|Vulnerabilities
+|The error list in Vulnerabilities are dependency errors that, after scanning packages and container images, are issues occurring as a part of your Supply Chain Security.
+
+|Secrets
+|A Secret is a programmatic access key that provides systems access to information, services, or assets. Secrets like API keys, encryption keys, OAuth tokens, certificates, PEM files, passwords, and passphrases are often explicitly stored in local or feature branches before being pushed to a main branch. In this view, see issues identified in your source code with remediation to perform a Manual Fix or Suppress the issue.
```
Comment (GiladMark) [inline code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan/monitor-fix-issues-in-scan.adoc]: we should mention this is shown in a file level

File: code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan/monitor-fix-issues-in-scan.adoc [hunk 4]
```diff
@@ -0,0 +1,373 @@
...
+
+You can also find relevant non-conformant scan results using filters or search using keywords or tags (if they are already a part of the code).
+For each misconfiguration on *Projects*, you can either Suppress the issue, Fix it from the Prisma Cloud console, or access the repository in the Version Control System (VCS) to perform a manual fix.
+Currently, this view of *Projects* is accessible on *Code Security > Projects > Enhanced*.
+
+image::proj-1.png[width=800]
+
+=== Resource and Policy Blocks
+
+All issues are viewable as a resource or policy block on *Projects* across code category views.
```
Comment (GiladMark) [inline code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan/monitor-fix-issues-in-scan.adoc]: policy block will be available by EOQ3

File: code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan/monitor-fix-issues-in-scan.adoc [hunk 5]
```diff
@@ -0,0 +1,373 @@
...
+
+* *Resource Block*
++
+After periodic scans on resources, Prisma Cloud generates contextualized scanned results of each resource as a resource block. Scan results are issues that are vulnerabilities or code errors found within the resource.
```
Comment (GiladMark) [inline code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan/monitor-fix-issues-in-scan.adoc]: This is relevant also for CI/CD and PR scans

File: code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan/monitor-fix-issues-in-scan.adoc [hunk 6]
```diff
@@ -0,0 +1,373 @@
...
+* *First Detected*: The timestamp of the issue when found.
+
+
+* *Secrets Resource Block*
++
+As a secrets issue, there is an extensive information in the resource block.
++
+image::proj-4.png[width=800]
++
+1. *Secret Name and Path*: Displays the repository name and it's code path.
```
Comment (GiladMark) [inline code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan/monitor-fix-issues-in-scan.adoc:186]: the "resource" here is a file so all secrets detected are in a file level and not repo

File: code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan/monitor-fix-issues-in-scan.adoc [hunk 7]
```diff
@@ -0,0 +1,373 @@
...
++
+2. *Total number of Issues*: Displays the total number of issues identified in the repository.
++
+3. *Additional Information*: Displays columns of the information regarding the issue.
++
+* *Secret type*: Displays the severity level of the exposed secret in the code.
++
+* *Risk Factors*: For Secrets there are three types of risk factors.
++
+** IsPublic: Identifies if the repository storing the secret is publicly accessible or private.
```
Comment (GiladMark) [inline code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan/monitor-fix-issues-in-scan.adoc]: Changed in the UI - Private repository/Public repository

File: code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan/monitor-fix-issues-in-scan.adoc [hunk 8]
```diff
@@ -0,0 +1,373 @@
...
+==== Resource Explorer
+
+The information on Resource Explorer enables you to make an educated decision on the security violation and understand if the violation has any connection as a dependency on other resources within the repository while exploring the change log of the resource.
+You can view this contextualized information across four tabs.
+
+* *Details*: Helps you understand the connection between resources while enabling you to make informed decisions if the connection is at risk or if it is necessary.
++
+image::proj-7.png[width=400]
+
+* *Errors*: Enables you to review security violations with the package severity threshold and utilize the information to either suppress or prioritize it.
```
Comment (GiladMark) [inline code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan/monitor-fix-issues-in-scan.adoc]: now it's Issues and it shows all statuses and not only Errors across all resource types

File: code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan/monitor-fix-issues-in-scan.adoc [hunk 9]
```diff
@@ -0,0 +1,373 @@
...
++
+image::proj-9.png[width=600]
+
+* *Traceability*: Explore and monitor connections between build-time and runtime resources.
+//+
+//image::[width=600]
++
+The support for History and Traceability is currently only IaC resources, and the support for Errors is currently only available for packages.
+
+==== Fix Cart
```
Comment (GiladMark) [inline code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan/monitor-fix-issues-in-scan.adoc:271]: UI is now changed - please update it on Tuesday before we release it

File: code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan/monitor-fix-issues-in-scan.adoc [hunk 10]
```diff
@@ -0,0 +1,373 @@
...
+===== Repositories
+
+A list of integrated repositories.
+
+image::proj-14.png[width=800]
+
+[#branch-]
+===== Branch
+
+A list of the supported branches of a VCS branch scan. Currently, the repository’s default branch is selected by default and cannot be configured
```
Comment (GiladMark) [inline code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan/monitor-fix-issues-in-scan.adoc]: this is relevant only for the first six default views(non- PR/CI-CD runs)

File: code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan/monitor-fix-issues-in-scan.adoc [hunk 11]
```diff
@@ -0,0 +1,373 @@
...
+|A resource appears with an error status when it is non-conformant to a policy.
+
+|Passed
+|A resource that has conformant policies or may have a history of fixed errors.
+
+|Suppressed
+|A resource previously appeared with a non-conformant policy but is suppressed with a Suppress action. To suppress a non-conformant policy in a resource is when you absolve the scanned result with a definitive explanation indicating the non-conformance to be not problematic.
+
+|Fix Pending
+|An issue awaiting remediation fix.
```
Comment (GiladMark) [inline code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan/monitor-fix-issues-in-scan.adoc]: *fix.. as part of a fix pull request that is not yet to merged

**Review by GiladMark — COMMENTED** [review]
Thanks @anaghapamidi , good job! 
We still left more comments under single comments + there is no section on all Filters we are currently support + the suppression flow + the manual flow

File: code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan/create-and-manage-code-category-views.adoc [hunk 12]
```diff
@@ -0,0 +1,40 @@
+:topic_type: task
+
+[.task]
+== Create and Manage Code Category views
+
+Prima Cloud provides eight default views of Overview, IaC misconfiguration, Vulnerabilities, Secrets, Licenses, Build Integrity, VCS Pull Requests and CI/CD Runs.
```
Comment (try-panwiac) [inline code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan/create-and-manage-code-category-views.adoc]: Can we add a value statement here? Something like

`Using views allows you to focus on use cases that are relevant to you.`

File: code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan/monitor-fix-issues-in-scan.adoc [hunk 13]
```diff
@@ -0,0 +1,373 @@
+== Monitor and Fix Issues in Your Scans
+
+toc::[]
+
+Prisma Cloud performs periodic scans on each integrated repository of the Version Control System (VCS) and CI/CD to find infrastructure misconfigurations.
```
Comment (try-panwiac) [inline code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan/monitor-fix-issues-in-scan.adoc]:
```suggestion
Prisma Cloud performs periodic scans on each integrated repository of onboarded Version Control Systems (VCS) and CI/CD pipelines to find infrastructure misconfigurations, open source vulnerabilities, license compliance violations, and exposed secrets.
```

We may need to highlight that VCS scans are periodic, CI/CD are on a trigger event

File: code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan/monitor-fix-issues-in-scan.adoc [hunk 14]
```diff
@@ -0,0 +1,373 @@
...
+
+*Projects* provides a consolidated view of scan results across your repositories, CI/CD and packages wherein categorization of issues are at resource or policy level giving you a better understanding on your security posture.
```
Comment (try-panwiac) [inline code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan/monitor-fix-issues-in-scan.adoc]:
```suggestion
*Projects* provides a consolidated view of scan results across your repositories and CI/CD pipelines wherein categorization of issues are at resource or policy level giving you a better understanding on your security posture.
```

File: code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan/monitor-fix-issues-in-scan.adoc [hunk 15]
```diff
@@ -0,0 +1,373 @@
...
+
+Prisma Cloud performs three types of scans on your integrations.
+
+* *VCS main branch scans*: Periodic scans performed on all main branches across repositories.
+* *VCS Pull Requests*: Periodic scans using Enforcement parameters are run on all open Pull Requests (PR) for your integrated repositories.
```
Comment (try-panwiac) [inline code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan/monitor-fix-issues-in-scan.adoc]: Not Periodic
```suggestion
* *VCS Pull Requests*: Event driven scans using Enforcement parameters are run on all open Pull Requests (PR) for your integrated repositories.
```

File: code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan/monitor-fix-issues-in-scan.adoc [hunk 16]
```diff
@@ -0,0 +1,373 @@
...
+* *CLI and CI/CD runs*: The Enforcement parameters with periodic scans are performed on all runs for your integrated repositories.
```
Comment (try-panwiac) [inline code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan/monitor-fix-issues-in-scan.adoc]:
```suggestion
* *CLI and CI/CD runs*: Event driven scans performed on runs as configured by the user using the Enforcement parameters.
```

File: code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan/monitor-fix-issues-in-scan.adoc [hunk 17]
```diff
@@ -0,0 +1,373 @@
...
+| Description
+
+|Overview
+|A summary view of all your scan results across all code categories. The Overview lists errors by prioritizing build issues across the integrations.
+
+|IaC Misconfiguration
+|The view lists security issues after scanning integrated repositories, wherein the master branch of a repository lists all code security violations in the resource block with code tags, resource dependencies and resource history. In this view, you can also identify Drift and trigger Manual Scans for your repository.
+
+|Vulnerabilities
+|The error list in Vulnerabilities are dependency errors that, after scanning packages and container images, are issues occurring as a part of your Supply Chain Security.
```
Comment (try-panwiac) [inline code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan/monitor-fix-issues-in-scan.adoc]:
```suggestion
|The error list in Vulnerabilities are open source dependency errors found in scanned open source packages and container image dependencies.
```

File: code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan/monitor-fix-issues-in-scan.adoc [hunk 18]
```diff
@@ -0,0 +1,373 @@
...
+
+|Secrets
+|A Secret is a programmatic access key that provides systems access to information, services, or assets. Secrets like API keys, encryption keys, OAuth tokens, certificates, PEM files, passwords, and passphrases are often explicitly stored in local or feature branches before being pushed to a main branch. In this view, see issues identified in your source code with remediation to perform a Manual Fix or Suppress the issue.
+
+|Licenses
+|The usage of open source software licenses composes most of the source codes. At the same time, there might be an oversight regarding the permission and rights to use or repurpose an open source code. In this view, see the listing of licensing issues with remediation solutions through periodic scans.
```
Comment (try-panwiac) [inline code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan/monitor-fix-issues-in-scan.adoc]:
```suggestion
|Most open source software includes a license governing its use. License scanning identifies packages that violate open source usage policies. In this view, see the listing of licensing issues through periodic scans.
```

File: code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan/monitor-fix-issues-in-scan.adoc [hunk 19]
```diff
@@ -0,0 +1,373 @@
...
+
+|Build Integrity
+|Repository misconfigurations occur when multiple owners for repositories exist on platforms like GitHub and GitLab. Through periodic scans, this view highlights the origin of another repository owner and prevents a chance Supply Chain attack.
```
Comment (try-panwiac) [inline code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan/monitor-fix-issues-in-scan.adoc]:
```suggestion
|Misconfigured repositories and CI/CD pipelines can lead to Supply Chain attacks such as data exfiltration and code injection. Use this view to spot VCS and CI/CD misconfigurations.
```

File: code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan/monitor-fix-issues-in-scan.adoc [hunk 20]
```diff
@@ -0,0 +1,373 @@
...
+
+|VCS Pull Requests
+| After configuring Enforcement, view issues for scans run on all open Pull Requests (PR) for your integrated repositories. On the Prisma Cloud console, you can access the scan result from the console or choose to access the VCS console and view the specific commit within PR for a manual fix.
+
+|CI/CD Runs
+|After configuring Enforcement, view issues for scans performing on runs of your integrated repositories. Using the open PR, you can access the specific run performed on the repository.
```
Comment (try-panwiac) [inline code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan/monitor-fix-issues-in-scan.adoc]:
```suggestion
|View issues for scans performing on runs of your integrated CI/CD pipelines.
```

File: code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan/monitor-fix-issues-in-scan.adoc [hunk 21]
```diff
@@ -0,0 +1,373 @@
...
+|Licenses
+|Secrets
+|Build Integrity
+
+|IaC Resource
+|✔️
+|✔️
+|✔️
+|
+|
```
Comment (try-panwiac) [inline code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan/monitor-fix-issues-in-scan.adoc:91]: @GiladMark is this true? I see only misconfigs in my view, but it's possible I missed CVEs

**Review by try-panwiac — COMMENTED** [review]
Added some thoughts

File: code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan/monitor-fix-issues-in-scan.adoc (hunk 18 above)
Comment (Pubs-MV) [reply code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan/monitor-fix-issues-in-scan.adoc]: Good edits @try-panwiac . Thank you!

File: code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan/fix-issues-in-a-scan-result.adoc (hunk 1 above)
Comment (anaghapamidi) [reply code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan/fix-issues-in-a-scan-result.adoc:30]: Added this section. @GiladMark

File: code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan/fix-issues-in-a-scan-result.adoc [hunk 22]
```diff
@@ -0,0 +1,72 @@
...
+
+=== Manual Fix an issue
+
+You can perform a manual fix for all issues. A manual fix enables you to access a specific commit to review the code to then resolve the issue manually using the policy guidelines on the Prisma Cloud console.
+
+image::proj-24.png[width=800]
+
+[.task]
+
+=== Suppress issues in a scan result
```
Comment (anaghapamidi) [inline code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan/fix-issues-in-a-scan-result.adoc:47]: Added the Suppress flow here @GiladMark

File: code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan/monitor-fix-issues-in-scan.adoc (hunk 14 above)
Comment (anaghapamidi) [reply code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan/monitor-fix-issues-in-scan.adoc]: Added @try-panwiac.

File: code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan/monitor-fix-issues-in-scan.adoc (hunk 13 above)
Comment (anaghapamidi) [reply code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan/monitor-fix-issues-in-scan.adoc]: Added @try-panwiac.

File: code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan/monitor-fix-issues-in-scan.adoc [hunk 23]
```diff
@@ -0,0 +1,372 @@
...
+
+All issues are viewable as a resource or policy block on *Projects* across code category views.
+
+* *Resource Block*
++
+After periodic scans on resources, Prisma Cloud generates contextualized scanned results of each resource as a resource block. Scan results are issues that are vulnerabilities or code errors found within the resource.
+Each resource block displays only five issues by default. *Show More* in each resource block helps you display more issues within the resource.
+
+* *Policy Block*
++
```
Comment (anaghapamidi) [inline code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan/monitor-fix-issues-in-scan.adoc]: Hidding this section as this is GA a few weeks from now. @GiladMark

File: code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan/fix-issues-in-a-scan-result.adoc [hunk 24]
```diff
@@ -0,0 +1,72 @@
...
+image::suppress.png[width=600]
++
+You can optionally add an *Expiration Time* for the suppression.
+
+. Select *Suppress by* to suppress issues based on the two suppression types.
++
+image::suppress-type.png[width=400]
++
+* *Resource*: This enables you to suppress the issues by resources and at your next scan these resources will not be scanned. You can also view the number of resources that will be affected to make an informed decision.
+* *Repositories*: This enables you to selectively suppress the issues across repositories. You are required to select the repositories from the list on the console.
```
Comment (GiladMark) [inline code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan/fix-issues-in-a-scan-result.adoc:68]: @anaghapamidi we do support more than this two suppressions. is there anything in our docs mentioning it?
if not - can we just refer it as an example for now and extend this content later on?

File: code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan/monitor-fix-issues-in-scan.adoc [hunk 25]
```diff
@@ -0,0 +1,371 @@
...
+|===
+
+| Projects Code Category View
+| Description
+
+|Overview
+|A summary view of all your scan results across all code categories. The Overview lists errors by prioritizing build issues across the integrations.
+
+|IaC Misconfiguration
+|The view lists security issues after scanning integrated repositories, wherein the default branch of a repository lists all code security violations in the resource block with code tags, resource dependencies and resource history. In this view, you can also identify Drift and trigger Manual Scans for your repository across any code categories view on the console.
```
Comment (GiladMark) [inline code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan/monitor-fix-issues-in-scan.adoc]: again - Manual Scans is not relevant for.a specific code category

File: code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan/monitor-fix-issues-in-scan.adoc [hunk 26]
```diff
@@ -0,0 +1,371 @@
...
+
+image::proj-1.png[width=800]
+
+=== Resource Blocks
+
+All issues are viewable as a resource on *Projects* across code category views.
+
+* *Resource Block*
++
+After periodic scans on resources, Prisma Cloud generates contextualized scanned results of each resource as a resource block. Scan results are are vulnerabilities in the code or code errors found within the resource.
```
Comment (GiladMark) [inline code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan/monitor-fix-issues-in-scan.adoc]: double "are"

File: code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan/monitor-fix-issues-in-scan.adoc [hunk 27]
```diff
@@ -0,0 +1,371 @@
...
+
+The information on Resource Explorer enables you to make an educated decision on the security violation and understand if the violation has any connection as a dependency on other resources within the repository while exploring the change log of the resource.
+You can view this contextualized information across four tabs.
+
+* *Details*: Helps you understand the connection between resources while enabling you to make informed decisions if the connection is at risk or if it is necessary.
++
+image::proj-7.png[width=400]
+
+* *Errors*: Enables you to review security issues across all resource types with the package severity threshold and utilize the information to either fix, suppress or manually add a fix to the issue.
++
```
Comment (GiladMark) [inline code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan/monitor-fix-issues-in-scan.adoc:258]: it called now Issues and it show all issues across multiple statuses (suppressed/passed/etc)

Comment (anaghapamidi) [reply code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan/monitor-fix-issues-in-scan.adoc:258]: On app-stage stage when I checked last it was still Errors. And currently the site is not stable. I will hide the image for now and fix the wording.

Comment (anaghapamidi) [reply code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan/monitor-fix-issues-in-scan.adoc:258]: Fixed the wording.

File: code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan/monitor-fix-issues-in-scan.adoc [hunk 28]
```diff
@@ -0,0 +1,371 @@
...
+
+Your scanned resources appear on *Code Security > Projects* with an active Error filter by default. You can choose to add more filters or remove the Error filter.
+
+[#severities-]
+===== Severities
+
+A Severities indicates an impact on a non-conformant resource in your repository. Resources can be filtered as Critical,High, Medium, Low and Informational in severity.
+
+image::proj-12.png[width=800]
+
```
Comment (GiladMark) [inline code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan/monitor-fix-issues-in-scan.adoc:348]: There are more filters that should be documented - see here https://docs.bridgecrew.io/docs/projects_january23#additional-filters

File: code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan/fix-issues-in-a-scan-result.adoc (hunk 24 above)
Comment (anaghapamidi) [reply code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan/fix-issues-in-a-scan-result.adoc:68]: @GiladMark - Please have a look here. Have added the rest.

File: code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan/monitor-fix-issues-in-scan.adoc (hunk 27 above)
Comment (anaghapamidi) [reply code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan/monitor-fix-issues-in-scan.adoc:258]: Fixed the wording.

File: code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan/monitor-fix-issues-in-scan.adoc (hunk 26 above)
Comment (anaghapamidi) [reply code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan/monitor-fix-issues-in-scan.adoc]: Fixed.

File: code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan/monitor-fix-issues-in-scan.adoc (hunk 23 above)
Comment (anaghapamidi) [reply code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan/monitor-fix-issues-in-scan.adoc]: Hidding this section as this is GA a few weeks from now. @GiladMark

File: code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan/monitor-fix-issues-in-scan.adoc (hunk 25 above)
Comment (anaghapamidi) [reply code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan/monitor-fix-issues-in-scan.adoc]: Fixed.

File: code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan/monitor-fix-issues-in-scan.adoc (hunk 28 above)
Comment (anaghapamidi) [reply code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan/monitor-fix-issues-in-scan.adoc:348]: Added the additional filters.

Comment (anaghapamidi) [reply code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan/monitor-fix-issues-in-scan.adoc:348]: Added the additional filters.

**Pull request merged**